styles/google-classroom/catppuccin.user.css

```
@@ -0,0 +1,1145 @@
+        .Mupove .xWw7yd:focus-within.h7Ww0,
+        .Mupove .xWw7yd :focus-within.JiTjnd,
+        .Mupove .xWw7yd :focus-within.h7Ww0 {
+            background-color: mix(@mauve, @base, 15%)
+        }
+
+        .Mupove .eumXzf:after {
+            border-color: @mauve;
+        }
+
+        /* tooltips */
+        .R8qYlc,
+        .gb_2e {
+            background-color: rgba(#rgbify(@surface0) [], 0.85)
+        }
+
+        .RM9ulf,
+        .gb_2e,
+        .gb_Ec div:first-child {
+            color: @text;
+        }
+
+        .gb_Ec > * {
+            color: @subtext0;
+        }
+
+
+        /*announce something to your class*/
+        .GWZ7yf {
+            box-shadow: none;
+            border: .0625rem solid @overlay0;
+        }
+
+
+        & when (@lookup =latte) {
+            .UvHKof.CNpREd.Fh209b .bFjUmb-Tvm9db:not(li .bFjUmb-Tvm9db) {
+                background-color: if(@overrideColors=1, @accent-color, @text)!important;
+            }
+
+            /*increase legibility*/
+            .m1PbN {
+                color: @crust!important;
+                fill: @crust!important;
+            }
+
+            .Aopndd,
+        .d4Fe0d,
+             .GWZ7yf, .BCUaBf:not(:empty), .hgjBDc{
+            background-color: @mantle;
+            border-color: @surface2;
+        }
+
+            .TGnLfc {
+                color: @crust!important;
+            }
+        }
+
+        & when not (@lookup =latte) {
+            /*completed assignment*/
+            .UvHKof .oC328b {
+                background-color: @surface0!important;
+            }
+
+            /*improve legibility of icons on assigned assignments when overriding colors*/
+            .m1PbN:not(.m1PbN.oC328b) {
+                color: if(@overrideColors=1, @crust, @text);
+            }
+        }
+    }
+
+
+    @media (prefers-color-scheme: light) {
+        :root {
+            #catppuccin(@lightFlavor, @accentColor);
+        }
+    }
+    @media (prefers-color-scheme: dark) {
+        :root {
+            #catppuccin(@darkFlavor, @accentColor);
+        }
+    }
+
+    /* If the website you are theming *does* have a light/dark mode toggle,
+     you can conditionally enable light/dark flavors by targeting whatever
+     attribute that website uses to indicate the theme. *The example below gives you an idea of what
+     to look for. The website you are themeing may use a class-based approach (e.g. :root.theme-dark),
+     or any of the myriad of other methods like this.*
+     Un-comment this block if that is the case.
+     For an actual example of this, see https://github.com/catppuccin/userstyles/blob/39288b593278e5efa48007d5fc001292cf9background672/styles/youtube/catppuccin.user.css#L21-L30.
+  :root[data-theme="dark"] {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+  :root[data-theme="light"] {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+  */
+}
+@-moz-document regexp('https?://drive.google.com/picker.*') {
```

you need to properly escape your regex
```suggestion
@-moz-document regexp('https?:\\/\\/drive\\.google\\.com\\/picker.*') {
```